pkg/aws/eni/routing/routing.go

```
@@ -42,6 +43,12 @@ var (
 // mtu: The ENI device MTU.
 // masq: Whether masquerading is enabled.
 func (info *RoutingInfo) Configure(ip net.IP, mtu int, masq bool) error {
+	if ip.To4() == nil {
+		log.WithFields(logrus.Fields{
+			"IP": ip,
+		}).Warning("Unable to Configure, the endpoint address is not an IPv4 address in ENI mode")
+		return nil
```

As mentioned, we should return an error here. Something like:

```
errors.New("malformed IP address") 
```